docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -80,6 +80,12 @@ In order to dynamically resolve it from request headers, you need to set the `ho
 
 For more details and more comprehensive scenarios, see https://www.keycloak.org/server/hostname[Configuring the hostname (v2)].
 
+== security-admin-console Client Redirect URIs
+
+In hostname v1 the admin URL was resolved dynamically from the request if the `hostname-admin` or `hostname-admin-url` options were not set. With hostname v2 the admin URL will default instead to the frontend URL. 
+If the `hostname` option is set and `hostname-strict` is true, this change will prevent redirect URIs with alternative hostnames from working as they did previously. 
+Instead you should consider using the `hostname-admin` option instead of the redirect URIs to allow a single alternative hostname.
```

Comment:
Just one thought. Should we perhaps provide some guidance for users tha customized their clients. We should probably mention which of the built-in clients are affected and what values needs to be reverted to what defaults.

Reply:
@vmuzikar the title of the section refers to the security-admin-console client, are there any other clients that are affected?

Reply:
Ah, yes, sorry, missed it's already in the title. No, I think it affects only this client.

Reply:
Ok, updated with more about reverting to the default redirect uris.